synapseclient/core/constants/concrete_types.py

```diff
@@ -10,6 +10,7 @@
 
 # Concrete types for FileHandles
 EXTERNAL_OBJECT_STORE_FILE_HANDLE = "org.sagebionetworks.repo.model.file.ExternalObjectStoreFileHandle"
+EXTERNAL_FILE_HANDLE = 'org.sagebionetworks.repo.model.file.ExternalFileHandle'
```

Comment:
Would it be preferable to be able to retrieve these via a rest call, or is maintaining this list here fine?

Reply:
These concrete types are spec’ed as part of the API so I think it’s fine to have them defined in client constants and I wouldn’t expect them to dynamically change any more than I would expect the endpoint urls to be dynamically change. If they were fetched with a remote call there would need to be a bootstrap request from the client.